db/migrate/20220913201800_add_user_ref_to_comments.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :user, null: false, foreign_key: true
+  end
+end
```

Comment:
- [ ] Same here, Please edit this migration because now it's creating a foreign key as `user_id` and It's supposed to be `author_id` like the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png).

Reply:
> * [ ]  Same here, Please edit this migration because now it's creating a foreign key as `user_id` and It's supposed to be `author_id` like the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png).

- [ ] As requested by the previous reviouwer 🙃  make sure the reference is created on the `author_id` column and not user_ide Kindly addrees this!! Inside you should a code similar to:

```ruby
  create_table :comments do |t|
    t.text :text
    t.timestamps # this will crate uopdated_at and created_at columns
    t.references :author, foreign_key: { to_table: 'users' } # this will create author_id and a corresponding index
    t.references :post, foreign_key: true # this will create post_id and a corresponding index (but you can also add column and index in separate commands as you did)
  end
```